components/expenses/Expense.js

```diff
@@ -23,6 +25,14 @@ import EditPayExpenseFeesForm from './EditPayExpenseFeesForm';
 import ConfirmationModal from '../ConfirmationModal';
 import StyledButton from '../StyledButton';
 
+const DeleteButton = styled(StyledButton)`
+  border: ${themeGet('colors.red.600')};
+  &:hover {
+    opacity: 0.8;
+    background: ${themeGet('colors.red.600')};
+  }
+`;
+
```

Comment:
A state for `danger` button style has been added to: https://www.figma.com/file/N4Xbl652BhzOutXmzhcrLGch/DS-06-%2F-Atoms?node-id=1928%3A7979. Let's add this to `lib/theme/index.js` and use the standard `StyledButton`.

![image](https://user-images.githubusercontent.com/1556356/69322966-1c48d280-0c46-11ea-83a0-ed500ec4ff06.png)